src/libraries/Microsoft.PowerFx.Core/Utils/DPath.cs

```diff
@@ -258,39 +206,23 @@ public string ToDottedSyntax(string punctuator = ".", bool escapeInnerName = fal
             for (var i = 0; i < Length; i++)
             {
                 sb.Append(sep);
-                var escapedName = escapeInnerName ? TexlLexer.EscapeName(this[i]) : this[i];
+                var escapedName = TexlLexer.EscapeName(this[i]);
                 sb.Append(escapedName);
-                sep = punctuator;
+                sep = TexlLexer.PunctuatorDot;
             }
 
             return sb.ToString();
         }
 
-        // Convert a path specified as a string to a DPath.
-        // Does not support individual path segments that contain '.' and '!' characters.
-        public static bool TryParse(string dotted, out DPath path)
```

Comment:
>TryParse [](http://example.com/codeflow?start=27&length=8)

Are we removing TryParse completely?

Reply:
yes, unused except by tests, and not a safe implementation.